app/controllers/v0/post911_gi_bill_statuses_controller.rb

```diff
@@ -1,25 +1,42 @@
 # frozen_string_literal: true
+require 'evss/gi_bill_status/gi_bill_status_response'
+
 module V0
   class Post911GIBillStatusesController < ApplicationController
     include SentryLogging
 
+    STATSD_GI_BILL_TOTAL_KEY = 'api.evss.gi_bill_status.total'
+    STATSD_GI_BILL_FAIL_KEY = 'api.evss.gi_bill_status.fail'
+
     def show
       response = service.get_gi_bill_status
-      if response.contains_education_info?
-        # 200
+      if response.success?
         render json: response,
                serializer: Post911GIBillStatusSerializer,
                meta: response.metadata
-      elsif response.evss_error?
+      else
+        error_type = response.error_type
+        StatsD.increment(STATSD_GI_BILL_FAIL_KEY, tags: ["error:#{error_type}"])
+        render_error_json(error_type)
+      end
+    ensure
+      StatsD.increment(STATSD_GI_BILL_TOTAL_KEY)
+    end
+
+    private
+
+    def render_error_json(error_type)
```

Comment:
Depending on the urgency of getting this PR in today this could be tech debt... Looking at this again I think it would be better to keep the controller 'skinny' and raise the errors from the service (then we'd log to sentry and raise in the same spot).